### DISCUSSION TITLE
Configuration sharing/resources

### DISCUSSION THREAD

**Discussion by Celticpure** [alf45tar/PedalinoMini#403] [General]
Hey everyone. I am just getting to grips with Pedalino it is really great. Is there a resource out there for sharing configuration files/setting that users can use? Would it be a useful addition to the repo?

**Comment by rigr** [comment]
I think sharing config files would be a good idea, BUT: the different versions of pedalino might cause problems, I am afraid. @alf45tar - what do you think - will the format of the config file stay the way it is now?

**Comment by alf45tar** [comment]
Different versions of PedalinoMini do not help to share configuration files but it the shared configuration it is based on Controls it can be reused by others just mapping Controls to Pedals.
